cfme/web_ui/tabstrip.py

```diff
@@ -119,6 +119,10 @@ def locate(self):
         select_tab(self.ident_string)
         return self.arg
 
+    def __getattr__(self, name):
```

Comment:
I think I don't get much of this step but that is maybe it is too late. Some lazy coercion?